app/assets/javascripts/components/timeline/timeline_handler.jsx

```diff
@@ -92,7 +92,6 @@ const TimelineHandler = createReactClass({
       key: 'wizard_handler',
       course: this.props.course,
       weeks: this.props.weeks,
-      week_meetings: weekMeetings,
```

Comment:
This just isn't used by any of the components that receive `courseProps`?

Reply:
I would say no. If I'm not mistaken, it goes only to the Meetings and Wizard and there is no use of week_meetings. I tried to double-check and now I can't see the point of passing meetings to that two components either.